graphics/VectorRendererSpec.cpp

```diff
@@ -761,17 +761,26 @@ template<typename PixelType>
 void VectorRendererSpec<PixelType>::
 blitKeyBitmap(const Graphics::ManagedSurface *source, const Common::Point &p, bool themeTrans) {
 	Common::Rect drawRect(p.x, p.y, p.x + source->w, p.y + source->h);
+	Common::Point np;
```

Comment:
Would suggest moving this declaration down to just above the if-else that assigns it, to keep that easier to keep track of.

Reply:
Moved down in commit b1d1c17.